### PR TITLE
upgrade apache version to AM26

### PR DESCRIPTION
**Pull request by utle**
Upgrade Apache version AM26

### CODE REVIEW

**Comment by Zech-Hein** [comment]
#build

**Comment by Zech-Hein** [comment]
build with z/os https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_UEodMH5yEe6pmuZkFgD-jA

**Comment by utle** [comment]
Manual PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_IzYoEH6WEe6pmuZkFgD-jA

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.3/build.gradle [hunk 1]
```diff
@@ -17,8 +17,8 @@
 def appBuildDir = "${buildDir}/test-application"
 
 dependencies { 
-    requiredLibs 'org.apache.directory.server:apacheds-all:2.0.0-M24',
-	'org.apache.directory.server:kerberos-client:2.0.0-M24',
+    requiredLibs 'org.apache.directory.server:apacheds-all:2.0.0.AM27',
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.3/build.gradle]:
```suggestion
    requiredLibs 'org.apache.directory.server:apacheds-all:2.0.0.AM26',
```

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.3/build.gradle [hunk 2]
```diff
@@ -17,8 +17,8 @@
...
+	'org.apache.directory.server:kerberos-client:2.0.0.AM27',
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5.3/build.gradle]:
```suggestion
	'org.apache.directory.server:kerberos-client:2.0.0.AM26',
```

**Comment by utle** [comment]
Manual PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_DOcagH_jEe6pmuZkFgD-jA

**Comment by Zech-Hein** [comment]
#build

**Pull request closed without merging**